cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/sriovdp-daemonset.yaml

```diff
@@ -100,13 +107,20 @@ spec:
       serviceAccountName: sriov-device-plugin
       containers:
       - name: kube-sriovdp
-        image: nfvpe/sriov-device-plugin:ppc64le
+        image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin:latest-ppc64le
```

Comment:
why do you use the latest if you already used `v3.4.0` above?

Reply:
Note that this manifest is taken from NPWG repo as is, any change to the manifest done by us is specified in the Kustomization.yaml file.